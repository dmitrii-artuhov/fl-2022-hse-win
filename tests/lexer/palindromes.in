start=ğŸ¤¯strğŸ¤¯
ğŸ¤¯strğŸ¤¯ ğŸ‘‰ ğŸ¥µağŸ¥µ ğŸ¤¯strğŸ¤¯ ğŸ¥µağŸ¥µ ğŸ¤Œ ğŸ¥µbğŸ¥µ ğŸ¤¯strğŸ¤¯ ğŸ¥µbğŸ¥µ ğŸ¤Œ ğŸ¥µağŸ¥µ ğŸ¤Œ ğŸ¥µbğŸ¥µ ğŸ¤Œ ğŸ˜µ ğŸ—¿