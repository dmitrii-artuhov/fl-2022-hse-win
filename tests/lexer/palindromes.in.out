LexToken(START,'str',1,0)
LexToken(NON_TERMINAL,'str',2,12)
LexToken(ARROW,'ðŸ‘‰',2,18)
LexToken(TERMINAL,'a',2,20)
LexToken(NON_TERMINAL,'str',2,24)
LexToken(TERMINAL,'a',2,30)
LexToken(SEPARATOR,'\U0001f90c',2,34)
LexToken(TERMINAL,'b',2,36)
LexToken(NON_TERMINAL,'str',2,40)
LexToken(TERMINAL,'b',2,46)
LexToken(SEPARATOR,'\U0001f90c',2,50)
LexToken(TERMINAL,'a',2,52)
LexToken(SEPARATOR,'\U0001f90c',2,56)
LexToken(TERMINAL,'b',2,58)
LexToken(SEPARATOR,'\U0001f90c',2,62)
LexToken(EMPTY,'ðŸ˜µ',2,64)
LexToken(END,'ðŸ—¿',2,66)
